prime-router/docs/getting-started/postgres-database.md

```diff
@@ -1,8 +1,266 @@
+[these Azure docs](https://docs.microsoft.com/en-us/azure/postgresql/concepts-read-replicas?msclkid=186f9575ac6d11eca07f9c72ead6d20d#failover-to-replica).
+
+
+#### Promote Read Replica to Read/Write
+
+NOTE: This action is not reversible, so be certain that you want to do this.
+
+1. In the Azure Portal, navigate to the PostgreSQL server that has read/write capabilities.
+2. In the `Replication` blade, click the `Stop Replication` button. This will promote the read-replica to a standalone
+   read/write instance.
+3. Update the connection strings for any resources requiring database access.
+4. NOTE: Performance may be degraded if your compute resources are in a different region than your database, so you may
+   need to adjust any alert settings.
+
+
+#### Provision New Server From Backup
+
+1. In the Azure Portal, navigate to the PostgreSQL server that has read/write capabilities.
+2. In the `Overview` blade, click the `Restore` button.
+3. Select the date and time to which to restore a new uniquely named server to.
+4. Click `Ok` to provision the new server from the point-in-time backup.
+
+## DB Flyaway Repair
+If the Flyway migrations need a repair, the following commands will resolve in each environment.
+
+## TEST
+
+```shell
+az login
+export FLYWAY_PASSWORD=$(az account get-access-token --resource-type oss-rdbms | python3 -c "import sys, json; print(json.load(sys.stdin)['accessToken'])")
+FLYWAY_URL=jdbc:postgresql://pdhtest-pgsql.postgres.database.azure.com:5432/prime_data_hub_candidate?sslmode=require FLYWAY_USER=reportstream_pgsql_admin@pdhtest-pgsql ./gradlew flywayRepair
+FLYWAY_URL=jdbc:postgresql://pdhtest-pgsql.postgres.database.azure.com:5432/prime_data_hub?sslmode=require FLYWAY_USER=reportstream_pgsql_admin@pdhtest-pgsql ./gradlew flywayRepair
+```
+
+## STAGING
+
+```shell
+az login
+export FLYWAY_PASSWORD=$(az account get-access-token --resource-type oss-rdbms | python3 -c "import sys, json; print(json.load(sys.stdin)['accessToken'])")
+FLYWAY_URL=jdbc:postgresql://pdhstaging-pgsql.postgres.database.azure.com:5432/prime_data_hub_candidate?sslmode=require FLYWAY_USER=reportstream_pgsql_admin@pdhstaging-pgsql ./gradlew flywayRepair
+FLYWAY_URL=jdbc:postgresql://pdhstaging-pgsql.postgres.database.azure.com:5432/prime_data_hub?sslmode=require FLYWAY_USER=reportstream_pgsql_admin@pdhstaging-pgsql ./gradlew flywayRepair
+```
+
+## PROD
+
+```shell
+az login
+export FLYWAY_PASSWORD=$(az account get-access-token --resource-type oss-rdbms | python3 -c "import sys, json; print(json.load(sys.stdin)['accessToken'])")
+FLYWAY_URL=jdbc:postgresql://pdhprod-pgsql.postgres.database.azure.com:5432/prime_data_hub_candidate?sslmode=require FLYWAY_USER=reportstream_pgsql_admin@pdhprod-pgsql ./gradlew flywayRepair
+FLYWAY_URL=jdbc:postgresql://pdhprod-pgsql.postgres.database.azure.com:5432/prime_data_hub?sslmode=require FLYWAY_USER=reportstream_pgsql_admin@pdhprod-pgsql ./gradlew flywayRepair
+```
+
+## How to Log and Debug Database Queries
```

Comment:
@thetaurean can you please review this section? I believe you're the original author? Any comments here? Should we add or remove content here?

Reply:
Talked to Gabriel and we are going to keep all methods in the doc. Removed the intro heading as well as the execution plan part.